src/components/cardbuilder/card.scss

```
@@ -871,3 +871,68 @@ button::-moz-focus-inner {
     transform: scale(1.4, 1.4);
     transition: 0.2s;
 }
+
+.cardRatingContainer {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    top: 0.3em;
+    padding: 0.3em 0.5em;
+    font-weight: 600;
+    background-color: rgba(0, 0, 0, 0.5);
+    transition: background-color 200ms ease-out;
+    border-radius: 1em;
+    z-index: 10;
+
+    span {
+        margin: 0 2px;
+    }
+
+    [dir="ltr"] & {
+        left: 0.3em;
+    }
+
+    [dir="rtl"] & {
+        right: 0.3em;
+    }
+}
+
+.card-hoverable:hover .cardRatingContainer {
+    background-color: rgba(0, 0, 0, 0);
+    transition: background-color 0;
+}
+
+.cardRatingContainer div:first-of-type ~ div {
+    margin-top: 0.3em;
+}
+
+.cardRating {
+    display: flex;
+    align-items: center;
```

```suggestion
    align-items: center;
    color: #fff;
```
The inherited color can make the ratings text hard to read on lighter themes